nearcore/src/config.rs

```diff
@@ -1493,12 +1493,10 @@ fn test_init_config_localnet() {
 /// correctly parsed and defaults being applied correctly applied.
 #[test]
 fn test_config_from_file() {
-    lazy_static_include::lazy_static_include_bytes! {
-        EXAMPLE_CONFIG_GC => "res/example-config-gc.json",
-        EXAMPLE_CONFIG_NO_GC => "res/example-config-no-gc.json",
-    };
+    let example_config_gc = std::fs::read("res/example-config-gc.json").unwrap();
+    let example_config_no_gc = std::fs::read("res/example-config-no-gc.json").unwrap();
 
-    for (has_gc, data) in [(true, &EXAMPLE_CONFIG_GC[..]), (false, &EXAMPLE_CONFIG_NO_GC[..])] {
+    for (has_gc, data) in [(true, &example_config_gc[..]), (false, &example_config_no_gc[..])] {
```

Comment:
Let’s put the `std::fs::read` inside of the loop and have paths in the iterator.